EventFilter/Utilities/plugins/RecoEventOutputModuleForFU.h

```diff
@@ -20,6 +20,18 @@
 
 
 namespace evf {
+
+  namespace {
+    struct flock make_flock(short type, short whence, off_t start, off_t len, pid_t pid)
```

Comment:
Tht `struct` is unnecesary.

Reply:
Instead of copying and pasting `make_flock`, how about moving it to a header file which can be included in both places it would now be needed?

Reply:
apparently it doesn't compile without struct. I found out that in case of 'stat' structure, struct is needed to resolve ambiguity with stat function. probably same here related to flock(2) function.